test_integration.sh

```diff
@@ -0,0 +1,282 @@
+
+# This is a test suite to run the SG and SG Accel unit tests against a live Couchbase Server
+# bucket (as opposed to walrus).  It works around the "interference" issues that seem to happen
+# when trying to run the full test suite (eg, running test.sh) as follows (maybe due to residual objects in Sync Gateway
+# memory like timer callbacks)
+#
+# Regarding maintenance, this might be a temporary script until the interference issues mentioned above
+# are solved.  However, if it does become a permanent script, then it should probably be upgraded to
+# leverage a tool like https://github.com/ungerik/pkgreflect to discover the list of tests.
```

Comment:
Do we have a ticket filed for the interference issues, with some details about what exactly is going on?  In addition to the maintenance issues, I expect it also takes longer to run in this mode?